### PR TITLE
Adding gcloud auth to fix the test

### PR DESCRIPTION
**Pull request by utka**
**What this PR does / why we need it**:
Proxy postsubmit is failing because we do not run bootstrap anymore which activated the account

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #https://github.com/istio/istio/issues/15714

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: prow/proxy-postsubmit.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ set -u
 # Print commands
 set -x
 
+gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
```
Comment (cjwagner) [inline prow/proxy-postsubmit.sh]: Use the `${GOOGLE_APPLICATION_CREDENTIALS}` env var instead of hard coding the location.

**Comment by cjwagner** [comment]
See also:
https://github.com/istio/test-infra/blob/5b1c7799eb6941b0efb6cc4c4486b5dc48276357/prow/bump.sh#L51-L55

**Comment by krzyzacy** [comment]
I think we still need to activate the service account explicitly, ref https://github.com/kubernetes/test-infra/issues/9471

**Comment by crazyxy** [comment]
Looks like the CI failed on all IPv6 related test.

**Comment by utka** [comment]
Looks like it is consistent failure, which started happening today.

**Pull request merged**